templates/phaser/packages/client/src/mud/createSystemCalls.ts

```diff
@@ -0,0 +1,21 @@
+import { getComponentValue } from "@latticexyz/recs";
+import { awaitStreamValue } from "@latticexyz/utils";
+import { ClientComponents } from "./createClientComponents";
+import { SetupNetworkResult } from "./setupNetwork";
+
+export type SystemCalls = ReturnType<typeof createSystemCalls>;
+
+export function createSystemCalls(
+  { worldSend, txReduced$, singletonEntity }: SetupNetworkResult,
+  { CounterTable }: ClientComponents
```

Comment:
I think we moved most things to `Counter` instead of `CounterTable`, should we do that here?

Reply:
ahh looks like the mud config and contracts are already using `Counter`, just need to regenerate client components (tsgen) and update this